src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/RSACng.EncryptDecrypt.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+using Microsoft.Win32.SafeHandles;
+
+using Internal.Cryptography;
+
+using ErrorCode = Interop.NCrypt.ErrorCode;
+using AsymmetricPaddingMode = Interop.NCrypt.AsymmetricPaddingMode;
+using BCRYPT_OAEP_PADDING_INFO = Interop.BCrypt.BCRYPT_OAEP_PADDING_INFO;
+
+namespace System.Security.Cryptography
+{
+    public sealed partial class RSACng : RSA
+    {
+        /// <summary>
+        ///     Encrypts data using the public key.
+        /// </summary>
+        public override byte[] Encrypt(byte[] data, RSAEncryptionPadding padding)
+        {
+            unsafe
```

Comment:
Nevermind; I see that the NCryptEncrypt/Decrypt functions being passed as delegates have pointers in their signatures.